fuzz/client.c

```diff
@@ -49,8 +49,8 @@ int FuzzerInitialize(int *argc, char ***argv)
     idx = SSL_get_ex_data_X509_STORE_CTX_idx();
     FuzzerSetRand();
     comp_methods = SSL_COMP_get_compression_methods();
-    sk_SSL_COMP_sort(comp_methods);
-
+    if (comp_methods != NULL)
```

Comment:
Without NULL check, could this be under no-comp compile time check?

#ifndef OPENSSL_NO_COMP
    comp_methods = SSL_COMP_get_compression_methods();
    sk_SSL_COMP_sort(comp_methods);
#endif

Reply:
Yes, it could be - and in fact I started out down that path. However `SSL_COMP_get_compression_methods()` can return `NULL` for other reasons (e.g. malloc failure), so actually I think the `NULL` check is required anyway.